### PR TITLE
[13.0][MIG] account_global_discount: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa TT26281

Main changes:
- Adapt code to v13.0 (account.invoice -> account.move, account.invoice.line -> account.move.line)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @sebastienbeau @chafique-delli please reconsider include the fixed amount inside this module instead of creating a new one, and check to make use of the same methods.

**Comment by pedrobaeza** [comment]
@ernestotejeda Travis is red

File: account_global_discount/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+To use this module, you need to:
+
+#. Go to *Settings > Parameters > Global Discounts*
+#. Add a new discount that can be either by percentage or fixed amount.
```
Comment (victoralmau) [inline account_global_discount/readme/USAGE.rst]: It is not possible to apply a fixed discount right? What is defined is only in % (it is for clarification and if appropriate change the USAGE.rst)

File: account_global_discount/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+            # Apply onchanges
+            tax_line._onchange_amount_currency()
+            tax_line._onchange_balance()
+
+    def _set_global_discounts_by_tax(self):
+        """Create invoice global discount lines by taxes combinations and
+        discounts.
+        """
+        self.ensure_one()
+        invoice_global_discounts = self.env["account.invoice.global.discount"]
```
Comment (victoralmau) [inline account_global_discount/models/account_move.py]:
```suggestion
```
IMO these line it's unnecessary.

**Review by victoralmau — APPROVED** [review]
Good job!! Only some little comments.

**Comment by pedrobaeza** [comment]
@sebastienbeau @chafique-delli have you checked this one? Do you think the fixed amount can be coupled here? I think so with not so many effort.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
As nobody more is commenting about this, let's merge it:

/ocabot merge nobump

**Pull request merged**